maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/test/java/org/apache/maven/scm/provider/git/jgit/command/remoteinfo/JGitRemoteInfoCommandTckTest.java

```diff
@@ -29,6 +29,8 @@
 import org.apache.maven.scm.provider.git.repository.GitScmProviderRepository;
 import org.eclipse.jgit.util.FileUtils;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
I don't understand why this import is added; it is the only change in this file?

Reply:
In junit3 you inherited this method (w/ same signature from TestCase), but in Junit4 you need to use hamcrest, or Assert or alike, hence static import but no other change (as method sig is same).